### PR TITLE
Simplified Views logic (task #4603)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

File: src/Template/Element/Menu/search-view-options.ctp [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Qobo Ltd. (https://www.qobo.biz)
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+
+$name = ($entity->get('name') ? $entity->get('name') : $this->name) . ' ' . date('Y-m-d H-m-s');
```
Comment (mamchenkov) [inline src/Template/Element/Menu/search-view-options.ctp]: @georgeconstantinou I think the date format is wrong here.

Comment (georgeconstantinou) [reply src/Template/Element/Menu/search-view-options.ctp]: @mamchenkov  this is for the export file name and this is how it looks: `Sites with posts 2017-10-13 08-10-03.csv`

Comment (mamchenkov) [reply src/Template/Element/Menu/search-view-options.ctp]: @georgeconstantinou I only mean the date format itself .. it uses 'm' for both minutes and months.

Comment (georgeconstantinou) [reply src/Template/Element/Menu/search-view-options.ctp]: @mamchenkov I see, fixed https://github.com/QoboLtd/cakephp-search/pull/176/commits/55e8c1ce979563f62fe198da61dde259d17ce4eb

**Pull request merged**